### ISSUE TITLE
Throttle only seems to obey overrides on the first comic

### ISSUE THREAD

**Issue by garbled1** [webcomics/dosage#198]
I tried setting a throttle for YAFGC, because it 520's after about 25 comics, but it didn't seem to be working, so I added a printf in RandomThrottle:

(venv) cbdl@rotanev:~$ dosage -b /comics/download -a YAFGC
YAFGC> Retrieving all strips
Throttle 3.005657434463501
YAFGC> Skipping existing file "/comics/download/YAFGC/2021-08-06-3469-Poetentially-a-clue.jpg".
Throttle 0.21980524063110352
Throttle 0.06767916679382324
YAFGC> Skipping existing file "/comics/download/YAFGC/2021-08-04-3468-a-most-singular-definition-of-normal.jpg".
Throttle 0.2181107997894287
YAFGC> Skipping existing file "/comics/download/YAFGC/2021-08-02-3467-just-one-little-thing.jpg".
Throttle 0.032395362854003906

I had set the throttle to 3, 15, so those super low values can't be right.

**Comment by TobiX** [comment]
That looks like the standard throttle is still in effect... It seems that comic redirects to `www.yafgc.net`, so you probably should correct the base URL to that, so the throttle applies to all requests uniformly:

```python
class YAFGC(_WordPressScraper):
    url = 'https://www.yafgc.net/'

    def __init__(self, name):
        super().__init__(name)
        self.session.add_throttle('www.yafgc.net', 3.0, 15.0)
```

**Comment by TobiX** [comment]
Seems to be solved with #199
